From eba1534eb3fb2d090657b4be9b0d5878dc3ede99 Mon Sep 17 00:00:00 2001
From: bholes <bholes@virtusa.com>
Date: Fri, 5 Feb 2016 18:17:29 +0530
Subject: [PATCH 4/5] finally worked

---
 webProject/build.gradle | 1 -
 1 file changed, 1 deletion(-)

diff --git a/webProject/build.gradle b/webProject/build.gradle
index f94b932..6386fad 100644
--- a/webProject/build.gradle
+++ b/webProject/build.gradle
@@ -18,5 +18,4 @@ compileJava.dependsOn unzip
 
 dependencies {
 	compile 'com.virtusa.app:myApp:1.0-SNAPSHOT'
-	compile 'com.truven.app:extApp:1.0-SNAPSHOT'
 }
-- 
1.9.5.msysgit.0

